examples/node/lib/rest/api/v2010/account/call/feedbackCallSummary.ts

```diff
@@ -319,7 +319,7 @@ export function FeedbackCallSummaryListInstance(
 
   instance._version = version;
   instance._solution = { accountSid };
-  instance._uri = `/Accounts/${accountSid}/Calls/Feedback/Summary.json`;
+  instance._uri = ``;
```

Comment:
Why does this get set to an empty string?

Reply:
The `ListInstance` object doesn't actually use it since it's a resource with no operations. We used to populate it, but it's not needed.